nc2zarr/converter.py

```diff
@@ -87,6 +89,7 @@ def __init__(self, *,
                  output_overwrite: bool = False,
                  output_append: bool = False,
                  output_append_dim: str = None,
+                 output_append_mode: str = "append_all",
```

Comment:
```suggestion
                 output_append_mode: AppendMode = AppendMode.append_all,
```

Reply:
Implemented along with corresponding necessary changes in commit cd5329b.